lib/dateTime.js

```diff
@@ -877,13 +877,18 @@ export function getDateTimeDescriptor() {
 			dateFormats = ['dddd, d MMMM yyyy', 'dd MMMM yyyy', 'dd/MM/yyyy', 'MMMM yyyy', 'd MMMM', 'd MMM'];
 			break;
 		case 'fr-ca':
-		case 'fr-on':
 			dateFormats[1] = 'MMM d yyyy';
 			dateFormats[2] = 'yyyy-MM-dd';
 			dateFormats[4] = 'MMMM d';
 			dateFormats[5] = 'MMM d';
 			firstDayOfWeek = 0;
 			break;
+		case 'fr-on':
+			dateFormats[0] = 'dddd\' le \'d MMMM yyyy';
+			dateFormats[1] = 'MMM d yyyy';
+			dateFormats[2] = 'yyyy-MM-dd';
```

Comment:
This is also going to change `dateFormats[4]` from `MMMM d` to `d MMMM` and `dateFormats[5]` from `MMM d` to `d MMM` -- did you intend to make that change as well?